docs/_plugins/search_index.rb

```diff
@@ -141,7 +141,13 @@ def benchmarking_results(post_url)
               row_data = {}
               values = line.split
               values.each_with_index do |value, j|
```

Comment:
Could we also check that the number of cells in the line matches the number of columns?

Reply:
Done